examples/spark/src/main/scala/org/apache/carbondata/examples/CaseClassDataFrameAPIExample.scala

```diff
@@ -42,8 +45,25 @@ object CaseClassDataFrameAPIExample {
       .mode(SaveMode.Overwrite)
       .save()
 
+    val employee = List(Employee("anubhav", 20000))
```

Comment:
I tried with `    val employee = List(Employee("anubhav", BigDecimal(324234353, 4)))`
the output is:
```
+-------+------+
|   name|salary|
+-------+------+
|anubhav| 32423|
+-------+------+
```
It is not handling correctly